### DISCUSSION TITLE
Too many exodus files while using an adaptive mesh

### DISCUSSION THREAD

**Discussion by jackeggemeyer** [idaholab/moose#28889] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hi all, 
I have a model that uses an adaptive mesh to create multiple exodus files per simulation. I have some custom kernels that don't have the best convergence but work, so I get a lot of time steps. I have noticed these files "corrupt" after 998 exodus files being created. 

Right now, I have to set my interval to a value and guess an end time so it doesn't exceed the 998 file limit. Is there a more elegant solution to this where MOOSE will stop after making 998 files? Or is it possible to have an adaptive mesh that only makes one exodus file? The size of the file will not be an issue. I have copied some of my code below. Thank you!

```
[Executioner]
  # Preconditioned JFNK (default)
  type = Transient
  nl_max_its = 15
  scheme = bdf2
  #solve_type = NEWTON
  solve_type = PJFNK
  petsc_options_iname = -pc_type
  petsc_options_value = asm
  l_max_its = 15
  l_tol = 1.0e-3
  nl_rel_tol = 1.0e-8
  start_time = 0.0
  # num_steps = 1
  end_time = 1450
  nl_abs_tol = 1e-11
  # dt = 1
  [./TimeStepper]
    type = IterationAdaptiveDT
        dt = 1
        optimal_iterations = 2
        iteration_window = 2
  [../]
  [./Predictor]
    type = SimplePredictor
    scale = 0.8
  [../]
[]

# [Debug]
#   show_var_residual_norms = true
# []

[Outputs]
    [./exodus]
      file_base = ../../Main/Data/10-18/testmpo3
      type = Exodus
      interval = 100
      sync_times = '1000 1200'
    [../]
  []
```

**Comment by GiudGiud** [comment]
Hello

If you know the end time, you can use the `simulation_time_interval` for that same purpose. 

I dont think it's normal this 998 problem. Do you have a minimal example showing the issue?

**Reply by GiudGiud** [reply]
You're right it's not!
Let me make a patch real quick

**Reply by GiudGiud** [reply]
use min_simulation_time_interval for now. I ll make sure we can output the final step too. Probably with a dedicated sync_time

**Reply by GiudGiud** [reply]
yes
you set
overwrite = True
and 
time_step_interval = 20
